packages/grid_client/scripts/applications/static_website.ts

```diff
@@ -0,0 +1,121 @@
+import { FilterOptions, GatewayNameModel, MachinesModel } from "../../src";
+import { config, getClient } from "../client_loader";
+import { log, pingNodes } from "../utils";
+
+async function deploy(client, vms, subdomain, gatewayNode) {
+  const resultVM = await client.machines.deploy(vms);
+  log("================= Deploying VM =================");
+  log(resultVM);
+  log("================= Deploying VM =================");
+
+  const vmPlanetary = (await client.machines.getObj(vms.name))[0].planetary;
+  //Name Gateway Model
+  const gw: GatewayNameModel = {
+    name: subdomain,
+    node_id: gatewayNode.nodeId,
+    tls_passthrough: false,
+    backends: ["http://[" + vmPlanetary + "]:9000"],
+  };
+
+  const resultGateway = await client.gateway.deploy_name(gw);
+  log("================= Deploying name gateway =================");
+  log(resultGateway);
+  log("================= Deploying name gateway =================");
```

Comment:
it'll be good if we moved this part to another function

Reply:
To follow the other scripts at https://github.com/threefoldtech/tfgrid-sdk-ts/pull/2743#discussion_r1607723148 and keep things consistent, we should leave it as it's.

I also added some comments for better reliability.

Reply:
indeed xD